controller/src/main/java/io/pravega/controller/server/ControllerService.java

```diff
@@ -195,6 +196,13 @@
                                          null);
     }
 
+    public CompletableFuture<ScaleStatusResponse> checkScale(final String scope, final String stream, final int epoch) {
```

Comment:
For API completion can we add a test to verify this throws Data not found exception if scope or stream is not found. Does it makes sense to explicitly check if epoch is not found? If its valid then it could be part of the API response so clients don't have to retry.